dts/bindings/bluetooth/nordic,nrf-dfe-ant.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# Note:
+#  All dfegpio[n]-pin properties have required property set to false. It
+#  depends on end user which Radio PSEL_GPIO[n] registers will be used to drive
+#  antenna switches.
+#  Nevertheless there is a requirement that at least two antennae are available.
+#  That means at least one dfegpio[n]-pin must be provided, to give Radio
+#  possibility to switch between these two antennae.
+
+description: |
+    Nordic DFE antennae matrix description
+
+compatible: "nordic,nrf-dfe-ant"
+
+include: [base.yaml]
+
+properties:
+    ant-num:
```

Comment:
AFAICT this property is unnecessary, as it can be calculated by the number of `dfegpio#-pin` properties that are valid.  I can't say offhand how to do that at compiletime, but I'm sure it's possible.  Otherwise just modify `radio_df_ant_num_get` to compute it from the pin array, maybe caching the value in a local `static uint8_t` if you expect to need it frequently.

Either way seems a better approach than having a property that might be inconsistent with the other properties.

Reply:
Perhaps values returned by `DT_NODE_HAS_PROP()` for all `dfegpioX-pin` properties could be just summed up?

Reply:
We can't get information about available antenna number from dfegpio#-pin e.g. to drive Nordic proprietary antenna matrix with 12 antennae you have to use 4 GPIO pins. 
The property is added to handle such situations.

Reply:
OK, that makes more sense.  A naive reader would assume each pin selects or deselects a single antenna.